todos/src/components/Todos.js

```diff
@@ -26,10 +32,15 @@ function Todos() {
               completedDate={elem.completedDate}
               onEdit={elem.onEdit}
             />
-          ) : null;
+          );
         })}
       </div>
-      {list.length === 0 && !isAddTaskVisible ? <EmptyTaskList /> : null}
+      {list.length === 0 && !isAddTaskVisible && <EmptyTaskList />}
```

Comment:
We can make our markup a lot cleaner by computing properties earlier in the function. It's one of those things dev often forget about is that, React components are just functions and they run after every render. So we can leverage that to compute value that depends on others. We can think about it as computed properties.

```suggestion
      const showEmptyListIcon = list.length === 0 && !isAddTaskVisible
      
      return(
       {showEmptyListIcon && <EmptyTaskList />}
      )
      
```

Reply:
Okay, @hhu-wtag Bhai. I will try to keep that in mind and write cleaner markups from now on. InShaAllah.